### ISSUE TITLE
Plans for this repo?

### ISSUE THREAD

**Issue by maricn** [romcok/nestjs-slonik#1]
Hi, there! I was just halfway writing my own slonik provider and integrating it in NestJS when I found this repo.

Do you plan to publish a package to npmjs, or at least add a license? Do you want to keep supporting the code, or is more like a one-time dump? :)

**Comment by cdesiles** [comment]
@maricn at the end did you end up with something?

**Comment by maricn** [comment]
> @maricn at the end did you end up with something?

@jokesterfr No, we went with deep integration in our project (not very complex; much alike the code here). Our implementation is not very extractable at this point, and the project is in icebox, so can't offer much help.

Good luck ✌️

**Comment by cdesiles** [comment]
Thanks for your answser! Will probably do the same, with a simple repository connector.
